bin/dev/kube-ready-state-check.sh

```diff
@@ -88,16 +88,22 @@ if having_category node ; then
     status "docker info should not show aufs"
 fi
 
-# kube-dns shows 4/4 ready
+# kube auth
+if having_category kube ; then
+    kubectl auth can-i get pods --namespace=kube-system 2>&1>/dev/null
```

Comment:
Is this intended to suppress stderr? Effectively it's just the same as:

```
kubectl auth can-i get pods --namespace=kube-system > /dev/null
```

If we want to suppress stderr and stdout, we can just use:
```
kubectl auth can-i get pods --namespace=kube-system &>/dev/null
```

Reply:
sure